AdblockPlusSafari/Assets.xcassets/Image.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Why is there Image.imageset in addition to trash.imageset? They seem to be about the same image.